packages/components/src/form-token-field/test/index.tsx

```diff
@@ -205,6 +206,45 @@ describe( 'FormTokenField', () => {
 			] );
 		} );
 
+		it( 'should add a token with the input value with onBlur when `tokenizeOnBlur` prop is `true`', async () => {
+			const user = userEvent.setup();
+
+			const onChangeSpy = jest.fn();
+
+			const { rerender } = render(
+				<FormTokenFieldWithState onChange={ onChangeSpy } />
+			);
+
+			const input = screen.getByRole( 'combobox' );
+
+			// Add 'grapefruit' token by typing it and check blur of field does not tokenize it.
+			await user.type( input, 'grapefruit' );
+			act( () => {
+				input.blur();
+			} );
```

Comment:
```suggestion
			await user.click( document.body );
```

I believe the current preference is to test as much as possible from the user's perspective. The suggested change above is how existing tests around blur in this file approach this. Making the approach consistent is probably a good thing too.

Reply:
Doh! That is the obvious solution that I couldn't see for looking this morning, thanks, updated in both places.